src/orchard/stacktrace.clj

```diff
@@ -0,0 +1,318 @@
+(ns orchard.stacktrace
+  "Cause and stacktrace analysis for exceptions"
+  {:added "0.31"
```

Comment:
Maybe you can expand this a bit, to be more in line with the level of high-level descriptions that most other namespaces have.